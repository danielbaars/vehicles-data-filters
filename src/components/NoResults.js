import React from 'react';

const NoResults = () => {
  return <div className='results__no-results'>Please pick a filter and the vehicles that have met your criteria will show up here...</div>;
};

export default NoResults;
