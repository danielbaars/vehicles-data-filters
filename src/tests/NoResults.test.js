import React from 'react';
import renderer from 'react-test-renderer';

import NoResults from '../components/NoResults';

describe('NoResults', () => {
  const component = <NoResults />;
  it('renders and matches our snapshot', () => {
    const rComponent = renderer.create(component);
    const tree = rComponent.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
