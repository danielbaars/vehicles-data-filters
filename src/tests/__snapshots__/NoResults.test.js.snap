// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResults renders and matches our snapshot 1`] = `
<div
  className="results__no-results"
>
  Please pick a filter and the vehicles that have met your criteria will show up here...
</div>
`;
